k8s/daemonsets/templates.jsonnet

```diff
@@ -314,4 +315,7 @@ local Experiment(name, index, bucket, datatypes=[]) = ExperimentNoIndex(name, da
 
   // The NDT tag to use for containers.
   ndtVersion: ndtVersion,
+
+  // The DASH tag to use for containers.
+  dashVersion: dashVersion,
```

Comment:
No need to put this here, just put the version directly in the daemonset.

The NDT version is here because it is used in both ndt and ndtcloud dameonsets.